server/pkg/utils/config/config.go

```diff
@@ -40,6 +40,7 @@ func ConfigureViper(environment string) error {
 	viper.SetEnvPrefix("ENTE")
 	// Ask Viper to look for underscores (instead of dots) for nested configs.
 	viper.SetEnvKeyReplacer(strings.NewReplacer(`.`, `_`))
+	viper.SetEnvKeyReplacer(strings.NewReplacer(`-`, `_`))
```

Comment:
Let's also add a comment above this, e.g.
```go
// Also replace "-" with underscores since "-" cannot be used in
// environment variable names.
viper.SetEnvKeyRepl...
```